bindings/node/src/mongocrypt.cc

```diff
@@ -674,6 +674,41 @@ NAN_METHOD(MongoCrypt::MakeDataKeyContext) {
         return;
     }
 
+    v8::Local<v8::Object> options = Nan::To<v8::Object>(info[1]).ToLocalChecked();
+    auto ALT_NAMES_KEY = Nan::New("keyAltNames").ToLocalChecked();
+    if (Nan::Has(options, ALT_NAMES_KEY).FromMaybe(false)) {
+        v8::Local<v8::Value> keyAltNames = Nan::Get(options, ALT_NAMES_KEY).ToLocalChecked();
+
+        if (!(keyAltNames->IsNull() || keyAltNames->IsUndefined())) {
```

Comment:
nit: can we replace this if statement with `if (!keyAltNames->IsArray()) { // bail with error }` instead of checking for !null, !undefined, isArray?

Reply:
Done! with slight change where if the input isn't an array it is ignored, the error handling down here is a duplication of what we do in the JS.